src/static/js/templates/prepare-worksheets/input-graded.hbs

```diff
@@ -0,0 +1,15 @@
+<section class="input-graded block__border-bottom u-mb20">
+  <div class="u-mt20 u-mb20">
+    <section class="input-with-btns">
+      <div class="input-with-btns_btns">
+        <button class="btn btn__grouped-first {{#if high_grade}}active{{/if}}">High</button>
+        <button class="btn btn__grouped {{#if medium_grade}}active{{/if}}">Med</button>
+        <button class="btn btn__grouped-last {{#if low_grade}}active{{/if}}">Low</button>
+        <span class="btn-delete cf-form_input-icon cf-icon cf-icon-delete-round" role="button" title="Delete"></span>
```

Comment:
I agree `<button>` would be preferable here. I did this because the design was in flux and using a button would mean I would have to override all the default styles (see https://github.com/cfpb/owning-a-home/pull/325#issuecomment-73053921). I wanted to keep the stylesheet at a minimum of customized overrides while developing, but yes, this should eventually be a button.

As a side note, I actually wonder if `<button>` and `<span role='button'>` are semantically equivalent? Do you know?